src/commands/new.ts

```diff
@@ -57,6 +64,31 @@ export default class New extends Command {
       });
     }
 
+    try {
+      const exampleFiles = await readFile(resolve(__dirname, '../../assets/examples/examples.json'), { encoding: 'utf8' });
+      examples = JSON.parse(exampleFiles);
+    } catch (error) {
+      // no examples found
+    }
+
+    if (examples.length > 0) {
+      questions.push({
+        name: 'use-example',
+        message: 'would you like to start your new file from one of our examples? (https://github.com/asyncapi/spec/tree/master/examples)',
```

Comment:
Can we make this message shorter?

Reply:
Yeah sure, any ideas? What do you think of having the link in there to show them what examples we are talking about, or not bothering? I'm in two minds about it....

Reply:
Yeah, not sure. Maybe just remove the link. Something we can do is show a message after an example is selected saying: "You can find more examples at https://github.com/asyncapi/spec/tree/master/examples".

Reply:
Cool thanks, will take a look mate

Reply:
Yeah, I just removed the URL for now completely going to keep it simple for now.